lib/cdo/script_constants.rb

```diff
@@ -111,6 +114,10 @@ module ScriptConstants
       ALGEBRA_A_NAME,
       ALGEBRA_B_NAME
     ],
+    research_studies: [
+      HOC_IMPACT_STUDY_NAME,
+      FLAPPY_IMPACT_STUDY_NAME
+    ],
```

Comment:
nit: Unclear if desired, but this could be inlined as
```
research_studies: [
  HOC_IMPACT_STUDY_NAME = 'hoc-impact-study'.freeze,
  FLAPPY_IMPACT_STUDY_NAME = 'flappy-impact-study'.freeze
],
```